ui/mod_user/user.py

```diff
@@ -0,0 +1,159 @@
+"""Database model declaration for a guild and a WoW guild."""
```

Comment:
I feel like the description is a bit lackluster, as this file is more supposed to be about users

Reply:
Done.